inventory/hosts

```diff
@@ -1,2 +1,3 @@
 [local]
-localhost region=ap-southeast-2 Owner="Development and Delivery" Division="Qld Online"
+# Amazon Linux 2023: al2023-ami-2023.4.20240611.0-kernel-6.1-x86_64
```

Comment:
can comment be updated to be 
``# ami-0e326862c8e74c0fe: Amazon Linux 2023: al2023-ami-2023.4.20240611.0-kernel-6.1-x86_64``

Reply:
Done. I have also updated to the latest Amazon Linux 2023 AMI.